src/sentry/models/group.py

```diff
@@ -133,6 +133,33 @@ class GroupStatus(object):
     MUTED = IGNORED
 
 
+# Statuses that can be queried/searched for
+STATUS_QUERY_CHOICES = {
+    "resolved": GroupStatus.RESOLVED,
+    "unresolved": GroupStatus.UNRESOLVED,
+    "ignored": GroupStatus.IGNORED,
+    # TODO(dcramer): remove in 9.0
+    "muted": GroupStatus.IGNORED,
```

Comment:
We should probably drop muted. Its been a while since 9.0.

Reply:
Want to avoid breaking people's searches in this PR. I think in theory we could use redash to figure out if people use this alias.

Reply:
```
select count(*) from sentry_savedsearch where query like '%is:muted%'
```

is greater 0 fwiw

Reply:
:cry: